File: helpers/solve.py
def solveInput(num1, sign, num2):
    if sign == '+':
        return num1 + num2
    elif sign == '-':
        return num1 - num2
    elif sign == '*':
        return num1 * num2
    elif sign == 'x':
        return num1 * num2
    elif sign == '/':
        return num1 / num2
    else:
        return 'not a valid sign'

File: jsbackup/helpers/solve.js
var solve = function solve(num1, sign, num2){
  switch(sign){
    case '+':
      return num1 + num2;
    break;

    case '-':
      return num1 - num2;
    break;

    case '*':
      return num1 * num2;
    break;

    case 'x':
      return num1 * num2;
    break;

    case '/':
      return num1 / num2;
    break;

    default:
      return 'not a valid sign';
    break;
  }
}

module.exports.solve = solve;